azure-pipelines/ui-automation/templates/flank/run-on-firebase-with-flank.yml

```diff
@@ -88,6 +91,7 @@ jobs:
               --environment-variables "clearPackageData=true" \
               --results-history-name "${{ parameters.resultsHistoryName }}" \
               --test-targets "${{ parameters.testTargetPackages }}" \
+              --test-targets "${{ parameters.ignoredTarget }} " \
```

Comment:
How does this work actually? Reading the line it seems the ignored are going to be included? but I believe they are actually left out?

Reply:
So it's a little weird, but the parameter passed in this case is "notAnnotation" (found in the yml files that are utilizing this template), which would exclude classes with a particular annotation, I don't think you can do 
`--test-targets " not${{ parameters.ignoredTarget }} "`

Reply:
Removed this, added it as part of default value for testTarget parameter.